content/en/docs/concepts/storage/volume-pvc-datasource.md

```diff
@@ -0,0 +1,68 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- thockin
+- msau42
+title: CSI Volume Cloning
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.15" state="alpha" >}} 
+This document describes the concept of cloning existing CSI Volumes in Kubernetes.  Familiarity with [Volumes](/docs/concepts/storage/volumes) is suggested.
+
+This feature requires VolumePVCDataSource feature gate to be enabled:
+
+```
+--feature-gates=VolumePVCDataSource=true
+```
+
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Introduction
+
+The {{< glossary_tooltip text="CSI" term_id="csi" >}} Volume Cloning feature adds support for specifying existing {{< glossary_tooltip text="PVC" term_id="persistent-volume-claim" >}}s in the `dataSource` field to indicate a user would like to clone a {{< glossary_tooltip term_id="volume" >}}.
+
+A Clone is defined as a duplicate of an existing Kubernetes Volume that can be consumed as any standard Volume would be.  The only difference is that upon provisioning, rather than creating a "new" empty Volume, the back end device creates an exact duplicate of the specified Volume.
+
+The implementation of Cloning, from the perspective of the Kubernetes API is the same as "creating volumes from snapshots", it leverages the DataSource field in the PVC spec.
+
```

Comment:
- possible edit: cloning instead of `Cloning`
- could you link to the page on `creating volumes from snapshots`?
- `dataSource` field